lxd/storage/backend_lxd.go

```diff
@@ -3466,7 +3466,9 @@ func (b *lxdBackend) CreateBucket(projectName string, bucket api.StorageBucketsP
 
 	revert.Add(func() { _ = BucketDBDelete(context.TODO(), b, bucketID) })
 
-	storageBucket := b.GetVolume(drivers.VolumeTypeBucket, drivers.ContentTypeFS, bucket.Name, bucket.Config)
+	bucketStorageName := project.StorageVolume(projectName, bucket.Name)
```

Comment:
Since we use `storageBucket` everywhere else, this should be renamed to `storageBucketName`.